.github/workflows/typescript-build-test-and-package.yml

```diff
@@ -18,8 +18,8 @@ jobs:
     env:
       NPM_TOKEN: ${{ secrets.npm-token }}
     steps:
-      - uses: actions/checkout@v2
-      - uses: actions/setup-node@v2
+      - uses: actions/checkout@v3
+      - uses: actions/setup-node@v3
```

Comment:
can we add yarn cache to speed up builds?
https://github.com/actions/setup-node#caching-global-packages-data

Reply:
done and done